### DISCUSSION TITLE
suggerimento per automazione con pulsante "virtuale"

### DISCUSSION THREAD

**Discussion by effe-7** [indomus/forum#2699] [HUB: Home Assistant]
non riesco ad arrivare a capo di come poter gestire un'esigenza pensata per come la gestivo con HOMEYPRO ma ovviamente non stò capendo come reinterpretarla con HASS.
vorrei avere un pulsante virtuale che all'accensione provveda ad accendere una serie di dispositivi e successivamente dopo un determinato tempo gli stessi dispositivi vengano spenti.

grazie

**Comment by 50494554524F** [comment]
button card + script + wait condition

**Reply by effe-7** [reply]
Creo il pulsante ma non riesco a richiamarlo nelle attivazioni

**Reply by snakuzzo** [reply]
leggi cosa ti ha scritto @50494554524F

**Reply by effe-7** [reply]
Si ho letto ma non capisco il.. button card

**Reply by effe-7** [reply]
possibile avere un esempio esplicito.
grazie
